build/wikia/transformers/polarities.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  Bar: 'naramon',
+  V: 'madurai',
+  D: 'vazarin',
+  U: 'umbra',
+  Ability: 'zenurik',
+  R: 'unairu'
```

Comment:
Missing aura polarity.
```suggestion
  R: 'unairu'
  Aura: 'aura'
```